openupgradelib/openupgrade.py

```diff
@@ -648,13 +660,11 @@ def m2o_to_x2m(cr, model, table, field, source_field):
     .. versionadded:: 8.0
     """
     if not model._columns.get(field):
-        raise orm.except_orm(
+        raise except_orm(
             "Error", "m2o_to_x2m: field %s doesn't exist in model %s" % (
                 field, model._name))
-    if isinstance(model._columns[field], (openerp.osv.fields.many2many,
-                                          Many2many)):
-        rel, id1, id2 = openerp.osv.fields.many2many._sql_names(
-            model._columns[field], model)
+    if isinstance(model._columns[field], (many2many, Many2many)):
+        rel, id1, id2 = many2many._sql_names(model._columns[field], model)
```

Comment:
My bad also for not thinking twice (it's sunday :stuck_out_tongue_winking_eye:). Fixed.